modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-overview.adoc

```diff
@@ -0,0 +1,11 @@
+[[overview]]
+== Overview
+
+The On-Premises version of the {pluginname} Converter is an application that can be installed and run on the customer’s in-house servers and computing infrastructure, including a private cloud. It contains all the features of the {pluginname} Converter available as SaaS.
+
+A valid license key is required in order to install {pluginname} Converter On-Premises.
+link:https://www.tiny.cloud/contact/[Contact us] for a trial license key.
+
+This section refers to a simplified version of the {pluginname} On-Premises which was designed to be easy to set up and maintain (while preserving all the features). It also lowers the running costs by reducing the number of servers required to run the whole application to the minimum.
```

Comment:
Remove this paragraph as it does not apply to us.

Reply:
```suggestion
```